src/Orckestra.Composer.Cart/Factory/Order/OrderDetailsViewModelFactory.cs

```diff
@@ -138,7 +138,8 @@ protected virtual OrderDetailInfoViewModel GetOrderInfosViewModel(CreateOrderDet
             orderInfos.OrderStatusRaw = param.Order.OrderStatus;
             orderInfos.BillingCurrency = param.Order.Cart.BillingCurrency;
             orderInfos.PricePaid = LocalizationProvider.FormatPrice((decimal)param.Order.Cart.Total, CurrencyProvider.GetCurrency());
-
+            orderInfos.IsOrderInEditableShipmentStatus =
+                param.EditableShipmentStatusSettings?.Contains(param.Order.OrderStatus) ?? false;
```

Comment:
extract  method protected virtual bool IsOrderEditable(... param)

and use it
orderInfos.IsOrderEditable = IsOrderEditable(param)